velox/exec/tests/StreamingAggregationTest.cpp

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include "utils/SumNonPODAggregate.h"
```

Comment:
This need to be a relative path from the root of repo

Reply:
done.